vaas/vaas/settings/base.py

```diff
@@ -201,12 +201,16 @@
 VCL_TEMPLATE_COMMENT_REGEX = env.str('VCL_TEMPLATE_COMMENT_REGEX', default=None)
 VCL_TEMPLATE_COMMENT_VALIDATION_MESSAGE = env.str('VCL_TEMPLATE_COMMENT_VALIDATION_MESSAGE', default=None)
 
-
-ROUTES_LEFT_CONDITIONS = env.dict('ROUTES_LEFT_CONDITIONS', default={
-    'req.url': 'URL',
-    'req.http.Host': 'Domain',
-    'req.http.X-Example': 'X-Example',
-})
+ROUTES_LEFT_CONDITIONS = {}
+if 'ROUTES_LEFT_CONDITIONS_BASE' in globals():
+    for i in ROUTES_LEFT_CONDITIONS_BASE:
```

Comment:
Common, its' not c++, let's do `for condition....`

Reply:
You have right!